docker/config/local/postgres/credentials.txt

```diff
@@ -0,0 +1,5 @@
+ENV POSTGRES_USER="conseiluser"
```

Comment:
between docker/config/local/conseil/conseil.conf, this, and docker/config/local/conseil/credentials.txt; it might make sense to keep those values in one place and reference them. (to the extent possible)

Reply:
I thought the same exact thing, however, Vishakh wanted the ability for the user to connect the conseil instance to a seperate db than the postgres instance in the repo.  Edge case?

Reply:
/local is for the fully integrated env. a user can always make /custom-env and make whatever changes they'd like there

Reply:
However, they would still need the ability to separately provide custom values for each application.